lms/static/sass/shared/_footer-v3.scss

```diff
@@ -0,0 +1,160 @@
+// Open edX: LMS footer
+// ====================
+@import '../base/grid-settings';
+@import 'neat/neat'; // lib - Neat
+
+
+// edX theme: LMS Footer
+// ====================
+$edx-footer-link-color: rgb(0, 158, 231);
+$edx-footer-bg-color: rgb(252,252,252);
+
+
+// Aggressively scoped for Drupal
+footer#footer-edx-v3 {
```

Comment:
Wow, aggressively scoped is right. Would you mind adding a comment to the top of this area to explain how this is being used and thus why it needs this sort of selector scope?